TOWebViewController/TOWebViewController.m

```diff
@@ -346,7 +346,7 @@ - (void)viewDidLoad
                                                                                         action:@selector(doneButtonTapped:)];
         }
         
-        self.doneButton.tintColor = self.buttonTintColor;
+        self.doneButton.tintColor = self.doneButtonTintColor;
```

Comment:
It might be more appropriate to check if `self.doneButtonTintColor` isn't `nil` first, else, just set it back to `self.buttonTintColor`. At the moment, I think this might break existing implementations since it'll be disabling the tint colour of 'Done' buttons.